notebook.ipynb

```diff
@@ -92,37 +92,1248 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "group_by_day = df_meteo_air_pollution.groupby(pd.Grouper(freq='D', key='date')).mean()\n",
+    "ws_pm10_corr = group_by_day[\"ws\"].corr(group_by_day[\"pm2.5\"])\n",
+    "print(ws_pm10_corr)\n",
+    "plt.scatter(group_by_day[\"ws\"], group_by_day[\"pm10\"])\n",
+    "plt.plot(np.unique(group_by_day[\"ws\"]), np.poly1d(np.polyfit(group_by_day[\"ws\"], group_by_day[\"pm10\"], 1))\n",
+    "         (np.unique(group_by_day[\"ws\"])), color='red')\n",
+    "plt.title('Correlation between PM10 and windspeed')\n",
+    "plt.xlabel('Windspeed in m/s')\n",
+    "plt.ylabel('particulate matter <10um in ug/m3')"
```

Comment:
The ylabel "<10um in ug/m3" is not really clear
Can you maybe change it to "Inhalable particles with diameter of 10 micrometers or smaller"?

Reply:
We have renamed the y label